build.gradle.kts

```diff
@@ -21,7 +21,9 @@ plugins {
 tasks {
     "lintKotlinMain"(LintTask::class) {
         exclude(
-            "**/*Generated.kt" // we can expand this list
+            "**/*Generated.kt",
+            "**/*Test.kt",
+            "**/Test*.kt" // we can expand this list
```

Comment:
I think we enable linter in this pr: #1515 
https://github.com/Flank/flank/commit/88b6c52a5e10cee38ce5931e7d9859fa3c1a70f8

Reply:
KK, something goes wrong during rebase